spec/views/articles_spec.rb

```diff
@@ -54,4 +54,14 @@ def create_comment(parent_id = nil)
     expect(rendered).to have_text("Originally published at")
     expect(rendered).to have_text("example.com")
   end
+
+  it "shows a note about the canonical URL after edit" do
+    allow(article1).to receive(:canonical_url).and_return("https://example.com/lamas")
+    allow(article1).to receive(:published_at).and_return(DateTime.strptime("1", "%s"))
+    allow(article1).to receive(:edited_at).and_return(DateTime.now)
```

Comment:
You should use `Time.current` instead of `DateTime.now`, check [this guide](http://danilenko.org/2012/7/6/rails_timezones/) on Rails and time